cmd/puppeth/wizard.go

```diff
@@ -160,27 +160,6 @@ func (w *wizard) readDefaultInt(def int) int {
 	}
 }
 
-// readFloat reads a single line from stdin, trimming if from spaces, enforcing it
-// to parse into a float.
-func (w *wizard) readFloat() float64 {
-	for {
-		fmt.Printf("> ")
-		text, err := w.in.ReadString('\n')
-		if err != nil {
-			log.Crit("Failed to read user input", "err", err)
-		}
-		if text = strings.TrimSpace(text); text == "" {
-			continue
-		}
-		val, err := strconv.ParseFloat(strings.TrimSpace(text), 64)
-		if err != nil {
-			log.Error("Invalid input, expected float", "err", err)
-			continue
-		}
-		return val
-	}
-}
```

Comment:
I'm unsure about this change. Although it's unused, it might be worthwhile to have it in there if someone needs it for the future. What's your take on it?